scripts/jinja/group.env

```diff
@@ -0,0 +1,2 @@
+GROUP=gingersnap
+USERS="['ryanemerson', 'pminz', 'tristantarrant', 'karesti', 'wburns', 'fax4ever', 'rigazilla', 'jabolina', 'pruivo', 'dpanshug']"
```

Comment:
@mimotej @HumairAK this is where https://github.com/mattrobenolt/jinja2-cli falls apart when parsing from ENV variables. I think yaml configs would work. 
Did you start looking into passing parameters from issues to the bot? Is there an example yaml config file?